src/mad_spark_alt/core/prompt_classifier.py

```
@@ -0,0 +1,435 @@
+                    "build", "implement", "code", "develop", "create", "design",
+                    "architecture", "API", "database", "algorithm", "system",
+                    "software", "app", "website", "platform", "framework",
+                    "programming", "technical", "engineering", "deployment",
+                    "infrastructure", "scalability", "performance", "security",
+                    # New additions
+                    "REST", "microservices", "docker", "kubernetes", "cloud",
+                    "backend", "frontend", "fullstack", "DevOps", "CI/CD",
+                    "testing", "debugging", "optimization", "refactor", "migrate",
+                    "integration", "authentication", "authorization", "encryption",
+                    "server", "client", "mobile", "web", "desktop", "embedded",
+                    "git", "version", "deploy", "monitor", "scale", "cache"
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)